Description: Fixes a compilation error (due to differences between ecj and javac?)
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: no
--- a/src/core/lombok/core/configuration/ConfigurationFile.java
+++ b/src/core/lombok/core/configuration/ConfigurationFile.java
@@ -273,7 +273,7 @@
 						ZipEntry entry = zip.getNextEntry();
 						if (entry == null) return null;
 						if (entry.getName().equals(file.getPath())) {
-							return read(zip);
+							return ConfigurationFile.read(zip);
 						}
 					}
 				} finally {
@@ -331,4 +331,4 @@
 			return null;
 		}
 	}
-}
\ No newline at end of file
+}
